Source/Fuse.Scripting.JavaScript/JavaScriptCore/JSTypedArrayInclude.h

```diff
@@ -1,11 +1,14 @@
-#include <Availability.h>
-#ifndef __MAC_10_12
-	#define __MAC_10_12 101200
-#endif
-#ifndef __IPHONE_10_0
-	#define __IPHONE_10_0 100000
-#endif
-#if __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0 || __MAC_OS_X_VERSION_MAX_ALLOWED >= __MAC_10_12
-	#include <JavaScriptCore/JSTypedArray.h>
-	#define JAVASCRIPTCORE_ARRAYBUFFER_SUPPORT
-#endif
+#ifdef ANDROID
```

Comment:
If at all possible, i.e. we're using a recent enough version of JSC on Android, we should define `JAVASCRIPTCORE_ARRAYBUFFER_SUPPORT`, as otherwise some operations on byte arrays and the like are very slow.

Reply:
If you're not sure, try it and see if the scripting tests pass.

Reply:
Sadly, scripting-tests is not in their best shape yet with this (nor with V8). I'm working on fixing what I can on master as we speak...

Reply:
Aha, I wasn't aware. [These tests](https://github.com/fusetools/fuselibs-public/blob/master/Source/Fuse.Scripting/Tests/Scripting.Test.uno) should be enough in this case. Are they also in bad shape?

Reply:
Sadly, yes. They crash on JSC context creation after some tests :(

Reply:
Ouch :(

Reply:
We get a linker error when trying to use the arraybuffersupport from android, so I'm guessing the version we pull from gradle is old.

I think we should (maybe as a seperate task) use this git@github.com:facebook/android-jsc.git to build jscore as an aar and then use https://stackoverflow.com/a/23326397/574033 to include it in our android builds.

Reply:
The problem with updating is that we don't have this option when cohabiting with RN.